client/app/admin/create-project/create-project.controller.js

```diff
@@ -11,9 +11,27 @@
 
     function createProjectController(mapService, drawService, projectService) {
         var vm = this;
+
+        // Wizard variables
         vm.currentStep = '';
+        vm.taskGrid = false;
```

Comment:
I'd usually prefix bools with is or are, eg

vm.isTaskGrid
vm.isTaskArbitrary